intel-sgx/dcap-artifact-retrieval/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "dcap-artifact-retrieval"
+version = "0.1.0"
+authors = ["Raoul Strackx <raoul.strackx@fortanix.com>"]
```

Comment:
Nit: worth adding @nshyrei to one of authors (authors field also could contain maintainers)